libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationMetricsWrapper.java

```diff
@@ -65,7 +74,10 @@ static void departEvent(SessionState sessionState, RouteProgress routeProgress,
       null, sessionState.currentStepCount(), sessionState.originalStepCount(),
       (int) routeProgress.distanceTraveled(), (int) routeProgress.distanceRemaining(),
       (int) routeProgress.durationRemaining(), sessionState.startTimestamp()
-    ));
+    );
+    Timber.d("Depart Location Engine name: " + sessionState.locationEngineName());
```

Comment:
Can we remove all the logging and remove the `Hashtable<String, Object> departEvent` returning it back to the original?